client/blocks/inline-help/placeholder-lines.jsx

```diff
@@ -1,9 +1,8 @@
-import { times } from 'lodash';
 import React from 'react';
 
 export default ( { lines = 4 } ) => (
 	<div className="inline-help__results-placeholder">
-		{ times( lines, ( n ) => (
+		{ Array.from( { length: lines }, ( _, n ) => (
 			<div key={ `line-${ n }` } className="inline-help__results-placeholder-item" />
```

Comment:
nit: the key can be just `n`, no need for a unique prefix.

Reply:
addressed in https://github.com/Automattic/wp-calypso/pull/56063/commits/1a5e55d29b039119fdb7068910e4f659e3c2e927